File: HW1/Homework1_CS/Form1.cs
using System;
using Microsoft.VisualBasic;

namespace Homework1_CS
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private int accumulator = 0;
        private string user = "guest";
     

        private void button3_Click(object sender, EventArgs e)
        {
            if (accumulator >= int.MinValue)
            {
                accumulator = accumulator - 1;
                label1.Text = accumulator.ToString();
            }
            else
                Interaction.MsgBox("Minumum number reached");
        }



        private void button2_Click(object sender, EventArgs e)
        {
            if (accumulator <= int.MaxValue)
            {
                accumulator = accumulator + 1;
                label1.Text = accumulator.ToString();
            }
            else
                Interaction.MsgBox("Max number reached");
        }

        private void button4_Click(object sender, EventArgs e)
        {
            Interaction.MsgBox("Goodbye " + user);
            System.Environment.Exit(0);
        }

        private void button5_Click(object sender, EventArgs e)
        {
            Interaction.MsgBox("Thank you!");
            user = textBox1.Text;
            textBox1.Enabled = false;
        }

        private void button1_Click(object sender, EventArgs e)
        {
                    }
    }
}

File: HW1/Homework_1-VBNET/Form1.vb
Public Class Form1
    Dim accumulator As Integer = 0
    Dim user As String = "guest"

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        If accumulator <= Integer.MaxValue Then
            accumulator = accumulator + 1
            Label1.Text = accumulator
        Else
            MsgBox("Max number reached")
        End If
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        If accumulator >= Integer.MinValue Then
            accumulator = accumulator - 1
            Label1.Text = accumulator
        Else
            MsgBox("Minumum number reached")
        End If
    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
        MsgBox("See you " + user)
        End
    End Sub

    Private Sub Button5_Click(sender As Object, e As EventArgs) Handles Button5.Click
        MsgBox("Thank you!")
        user = TextBox1.Text
        TextBox1.Enabled = False
    End Sub

    Private Sub TextBox1_TextChanged(sender As Object, e As EventArgs) Handles TextBox1.TextChanged

    End Sub
End Class
